js/Controller/Event/Event.js

```diff
@@ -0,0 +1,49 @@
+import Product from '../../Model/Product/Product.js';
+import ProductManageMenu from '../../View/Template/ProductManageMenu.js';
+import ProductPurchaseMenu from '../../View/Template/ProductPurchaseMenu.js';
+import VendingMachineManageMenu from '../../View/Template/VendingMachineManageMenu.js';
+import Employee from '../Controller/Employee.js';
+
+const Event = (function () {
+  const employee = Employee.of();
+
+  return {
+    router: {
+      click(event) {
```

Comment:
이건 취향에 따른 부분일수도 있는데 아래처럼 구조분해 할당을 통해 사용할 값만 가져와도 좋겠다는 생각이 드네용
```suggestion
      click({ target: { id }}) {
```